lib/embed/embed.dart

```diff
@@ -96,6 +96,25 @@ class PlaygroundMobile {
     });
   }
 
+  /**
+   * Return if strong mode should be enabled
+   */
+  bool _isStrongMode(String strong) {
```

Comment:
Ah! I think we'll need to move some bits of Luke's PR (https://github.com/dart-lang/dart-pad/pull/731) into here: https://github.com/dart-lang/dart-pad/blob/master/lib/embed/embed.dart. If I recall correctly, embed.dart is a modified copy of playground.dart.